packages/cli/src/utils/config.ts

```diff
@@ -23,10 +24,19 @@ export function getConfig(resave = true): I18nConfig | null {
   return result;
 }
 
-export function saveConfig(config: I18nConfig) {
+export async function saveConfig(config: I18nConfig) {
   const configFilePath = _getConfigFilePath();
 
-  const serialized = JSON.stringify(config, null, 2);
+  const prettierConfigFilePath = await prettier.resolveConfigFile() || ''
+
+  const prettierOptions = await prettier.resolveConfig(prettierConfigFilePath, {editorconfig: true}) || {
+    parser: 'json',
+    useTabs: false,
+    tabWidth: 2,
+  }
+    
+  const serialized = await prettier.format(JSON.stringify(config), prettierOptions);
+  
```

Comment:
tbh this looks really good. how do we know it's working just looking at the code? be sure to add tests that showcase the core logic actually works here, and let's merge. good stuff!